src/react/utils.js

```diff
@@ -775,20 +775,26 @@ export function convertConfigObjectToReactComponentTreeConfig(
   // defaults
   let firstRenderOnly = false;
   let isRoot = false;
+  let modelString;
 
   if (!(config instanceof UndefinedValue)) {
     for (let [key] of config.properties) {
       let propValue = getProperty(realm, config, key);
       if (propValue instanceof StringValue || propValue instanceof NumberValue || propValue instanceof BooleanValue) {
         let value = propValue.value;
 
-        // boolean options
         if (typeof value === "boolean") {
+          // boolean options
           if (key === "firstRenderOnly") {
             firstRenderOnly = value;
           } else if (key === "isRoot") {
             isRoot = value;
           }
+        } else if (typeof value === "string") {
+          // string options
+          if (key === "model") {
+            modelString = value;
```

Comment:
You probably want to do some sort of input validation here to make sure that the string parses and won't throw an unhandled error later. You can probably just grab the logic from `__optimize`.